src/cloud/project/new-relic.md

```diff
@@ -149,13 +178,17 @@ To remove a license key from an environment:
    magento-cloud variable:delete php:newrelic.license
    ```
 
-## Investigate performance
+## Using New Relic
```

Comment:
The previous level 2 headings were Set up and Connect. Maybe this should be something similar? Use?